algorithms/src/crh/bowe_hopwood_pedersen.rs

```diff
@@ -16,25 +16,23 @@
 
 pub use crate::crh::pedersen_parameters::PedersenSize;
 
+use super::bowe_hopwood_pedersen_parameters::*;
 use crate::{
     crh::{PedersenCRH, PedersenCRHParameters},
     errors::CRHError,
     traits::CRH,
 };
+use bitvec::{order::Lsb0, view::BitView};
 use snarkvm_curves::Group;
 use snarkvm_fields::{ConstraintFieldError, Field, PrimeField, ToConstraintField};
-use snarkvm_utilities::{biginteger::biginteger::BigInteger, bytes_to_bits};
+use snarkvm_utilities::biginteger::biginteger::BigInteger;
 
 use rand::Rng;
 
-#[cfg(feature = "parallel")]
-use rayon::prelude::*;
-
-pub const BOWE_HOPWOOD_CHUNK_SIZE: usize = 3;
-
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
 pub struct BoweHopwoodPedersenCRH<G: Group, S: PedersenSize> {
     pub parameters: PedersenCRHParameters<G, S>,
```

Comment:
Do we still need these parameters? It appears `bowe_hopwood_parameters` are the only used params here. Perhaps we can remove the unused `parameters and rename `bowe_hopwood_parameters` to `parameters` .

Reply:
They are necessary for serialization as `bowe_hopwood_parameters` are just a view of `parameters`.